video_xblock/static/html/brightcove.html

```diff
@@ -9,6 +9,7 @@
         width="100%"
         height="560"
         class="video-js"
+        brightcove="true"
```

Comment:
If I recall correctly, `="true"` is redundant. See samples - https://html.spec.whatwg.org/#boolean-attributes